server/client_test.go

```diff
@@ -1735,6 +1735,74 @@ func TestTraceMsg(t *testing.T) {
 	}
 }
 
+func TestTraceMsgHeadersOnly(t *testing.T) {
```

Comment:
@alexbozhenko Here is the new test. When `TraceHeaders` is true, it will strip out the payload of the message. You can see the first table test results in the empty string (since it is not logged at all). Where as the third test that includes a header and payload is still logged, but the payload is stripped.

Reply:
thank you, @bruth !